### ISSUE TITLE
Blank screen on all websites.

### ISSUE THREAD

**Issue by ItzzCode** [ruffle-rs/ruffle#3636]
The bug appears trying to play any flash game on my computer, as a blank screen will appear trying to play anything.

Papa's Donuteria
![image](https://user-images.githubusercontent.com/51547168/111036594-f1136500-83ed-11eb-8b70-f2df66200fb1.png)

I'm using Ruffle to enjoy flash games, so there should be games. This is happing on the web extension with nightly 2021-3-12 and 2021-3-13 on Chrome for Windows 10.

**Comment by danielhjacobs** [comment]
Websie: http://www.flipline.com/games/papasdonuteria/index.html

JavaScript:

```
Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
index.html:1 Access to fetch at 'http://i.flipline.com/gamefiles/papasdonuteria/papasdonuteria_102.swf' from origin 'http://www.flipline.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
i.flipline.com/gamefiles/papasdonuteria/papasdonuteria_102.swf:1 Failed to load resource: net::ERR_FAILED
VM39:3 Asynchronous error occurred: Could not fetch movie http://i.flipline.com/gamefiles/papasdonuteria/papasdonuteria_102.swf
```

Likely a duplicate of #2201?

**Comment by Poopooracoocoo** [comment]
@ItzzCode To be clear, do the samples on https://ruffle.rs/demo work for you?

**Comment by ItzzCode** [comment]
@Poopooracoocoo No, the demos do not work either.

**Comment by ItzzCode** [comment]
> CORS policy related. Perhaps a duplicate of, or if not at least similar to, #2201?

Also, what's CORS, I'm not very good at these things.

**Comment by danielhjacobs** [comment]
CORS: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS. If the demo animations aren't working though, you probably have another problem as well. However, even if you fix that first issue, that particular animation will likely no longer work. 

Basically, Ruffle uses something called the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) to load animations. It uses an underlying [WebAssembly](https://developer.mozilla.org/en-US/docs/WebAssembly) file, which requests the swf file, interprets it, and translates it into WebAssembly using the Rust programming language.

However, if the swf is hosted on a different site than the one where the HTML page is located, a CORS error occurs. Embedded animations were not reliant on CORS (back when they worked), but the Fetch API is.

**Comment by Poopooracoocoo** [comment]
@ItzzCode hey are you still unable to run the samples on the Ruffle demo? If so, reopen your issue or make a new one :) 

Daniel was explaining that Papa's Donuteria wouldn't work anyway. Ruffle's samples should however. 
What are the specs of your computer? 
What version of Chrome?
